content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -70,15 +79,26 @@ const ActivityFeedItem = React.createClass({
         </div>
       </a>
       <div className="action-items-container">
-        <div className="action-item icon-delete" ref="delete" onClick={() => this.props.onDelete(site.url)}></div>
```

Comment:
I see that `getDefaultProps()` still has an `onDelete` placeholder. Can remove now?